x/metadata/types/events.go

```diff
@@ -22,26 +22,26 @@ const (
 	TKObjectAction string = "object-action"
 
 	// TLCategory is a string name for labels defining an object category.
-	TLCategory               string            = "category"
-	TLCategory_Entry         TelemetryCategory = "entry"
-	TLCategory_Specification TelemetryCategory = "specification"
-	TLCategory_OSLocator     TelemetryCategory = "object-store-locator"
+	TLCategory              string            = "category"
+	TlcategoryEntry         TelemetryCategory = "entry"
+	TlcategorySpecification TelemetryCategory = "specification"
+	TlcategoryOslocator     TelemetryCategory = "object-store-locator"
 
 	// TLType is a string name for labels defining an object type.
-	TLType              string              = "object-type"
-	TLType_Scope        TelemetryObjectType = "scope"
-	TLType_Session      TelemetryObjectType = "session"
-	TLType_Record       TelemetryObjectType = "record"
-	TLType_ScopeSpec    TelemetryObjectType = "scope-specification"
-	TLType_ContractSpec TelemetryObjectType = "contract-specification"
-	TLType_RecordSpec   TelemetryObjectType = "record-specification"
-	TLType_OSLocator    TelemetryObjectType = "object-store-locator"
+	TLType             string              = "object-type"
+	TltypeScope        TelemetryObjectType = "scope"
+	TltypeSession      TelemetryObjectType = "session"
+	TltypeRecord       TelemetryObjectType = "record"
+	TltypeScopespec    TelemetryObjectType = "scope-specification"
+	TltypeContractspec TelemetryObjectType = "contract-specification"
+	TltypeRecordspec   TelemetryObjectType = "record-specification"
+	TltypeOslocator    TelemetryObjectType = "object-store-locator"
 
 	// TLAction is a string name for labels defining an action taken.
-	TLAction         string          = "action"
-	TLAction_Created TelemetryAction = "created"
-	TLAction_Updated TelemetryAction = "updated"
-	TLAction_Deleted TelemetryAction = "deleted"
+	TLAction        string          = "action"
+	TlactionCreated TelemetryAction = "created"
+	TlactionUpdated TelemetryAction = "updated"
+	TlactionDeleted TelemetryAction = "deleted"
```

Comment:
The `a` in "action" should be capitalized for all these, e.g. `TlActionCreated`.